uvtrick/__init__.py

```diff
@@ -104,24 +105,26 @@ def run(self, func: Callable, *args, **kwargs):
             output = temp_dir / "tmp.pickle"
             # First pickle the inputs
             inputs.write_bytes(pickle.dumps((args, kwargs)))
+
             # Now write the contents of the script
             contents = textwrap.dedent(inspect.getsource(func))
             contents += "\n\n"
             contents += maincall(func, temp_dir / PICKLED_INPUTS_PATH, temp_dir / PICKLED_OUTPUTS_PATH)
             script.write_text(contents)
             
-            deps = " ".join([f"--with {dep}" for dep in self.requirements])
-            pyversion = f"--python {self.python}" if self.python else ""
-            quiet = "--quiet" if not self.debug else ""
+            quiet = [] if self.debug else ["--quiet"]
+            deps = [f"--with={dep}" for dep in self.requirements]
+            pyversion = [f"--python=self.python"] if self.python else []
```

Comment:
```suggestion
            pyversion = [f"--python={self.python}"] if self.python else []
```

Reply:
Fixes this one manually, @lmmx seems like a forgotten f-string here.

Reply:
Oops, apologies: looks like that calls for test coverage...